### PR TITLE
Bgrenon manageprojects refresh

### PR DESCRIPTION
**Pull request by bgrenon**
https://neon-next-git-bgrenon-manageprojects-refresh-neondatabase.vercel.app/docs/manage/projects

For reviewers:

- Please check the new page re-organization. Hopefully navigation is a bit easier, there is a lot on this page.
- In the UI, is the text "1 shared vCPU..." misleading? Would it be more accurate to say ".5 share of 1 vCPU..."?
image

- We don't explain what the 1/4 and 1/2 shared compute means - who or what is it shared with?
- On the Settings -> General page, the ID is copyable but not editable. Can you confirm that the ID is permanent and unchangeable?
- Please check the [Reset compute size section](https://neon-next-git-bgrenon-manageprojects-refresh-neondatabase.vercel.app/docs/manage/projects#reset-the-default-compute-size), there is some expanded intro content there. Also added a new annotated screenshot -- open to any comments on that!

### CODE REVIEW

File: content/docs/manage/projects.md [hunk 1]
```diff
@@ -2,82 +2,93 @@
...
-## Create a project
+## About the Settings page
+
+Once you open a project, you can use the **Settings** page to manage that particular project and configure any defaults.
+
+![Project Settings page](/docs/manage/settings_page.png)
+
+You can tab between these sections:
+
+- **General** &#8212; Change the name of your project or copy ID.
```
Comment (danieltprice) [inline content/docs/manage/projects.md:37]:
```suggestion
- **General** &#8212; Change the name of your project or copy the project ID.
```

File: content/docs/manage/projects.md [hunk 2]
```diff
@@ -112,6 +123,47 @@ The costs associated with a shared project are charged to the Neon account that
...
+To reset the default compute size, go to **Settings** > **Compute**.
+
+You can choose from two options:
+
+- **Fixed Size:** Select a fixed compute size ranging from .25 CUs to 7 CUs. A fixed-size compute does not scale to meet workload demand.
+- **Autoscaling:** This option allows you to specify a minimum and maximum compute size. Neon scales the compute size up and down within the selected compute size boundaries in response to the current load. Currently, _Autoscaling_ supports a range of 1/4 (.25) CU to 7 CUs. The 1/4 CU and 1/2 CU settings are _shared compute_. For information about how Neon implements the _Autoscaling_ feature, see [Autoscaling](/docs/introduction/autoscaling).
+
+_Example: default minimum and maximum autoscale settings_
+
+![Default autoscaling min and max](/docs/manage/default_autoscale.png)
```
Comment (danieltprice) [inline content/docs/manage/projects.md:149]: Not sure why, but the items selected in the sidebar in the image are hard to see.

**Comment by danieltprice** [comment]
I can confirm that the project ID is not editable. It's a generated value. Good question about shared vCPU. Here's what everyone's favorite AI assistant has to say:

"A shared vCPU refers to a virtual CPU (vCPU) where the physical processing power (from the actual CPU in a computer) is distributed or shared among multiple virtual machines (VMs) or tasks. Instead of dedicating an entire physical CPU core to one VM, the computing resources are split among several VMs."

If that's accurate, maybe we can add a glossary term to our docs with a similar definition.

**Review by danieltprice — APPROVED** [review]
A few minor comments, but otherwise, LGTM!

File: public/docs/manage/settings_page.png
[the remark this answers is not in the document]
Comment (bgrenon) [reply public/docs/manage/settings_page.png:1]: Will keep an eye out for this one as well.

File: content/docs/manage/projects.md [hunk 3]
```diff
@@ -2,82 +2,93 @@
...
+## About the Settings page
+
+Once you open a project, you can use the **Settings** page to manage that particular project and configure any defaults.
+
+![Project Settings page](/docs/manage/settings_page.png)
+
+You can tab between these sections:
+
+- **General** &#8212; Change the name of your project or copy ID.
+- **Compute** &#8212; Set the default sizes for any new compute instances you create when branching.
```
[the remark this answers is not in the document]
Comment (bgrenon) [reply content/docs/manage/projects.md:38]: Changed to:

"Set the auto-suspend and sizing defaults for any new compute instances you create when branching."

**Pull request merged**